packages/graphql/src/entities/App.ts

```diff
@@ -12,6 +12,13 @@ export class App {
 
   constructor (private ctx: BaseContext) {}
 
+  @nxs.field.nonNull.string({
+    description: 'See if the GraphQL server is alive',
+  })
+  get healthCheck (): NxsResult<'App', 'healthCheck'> {
```

Comment:
Just added for testing but I think it's fine to leave this for now. Can remove if we don't want to include it, but it might be useful to have.